pkgs/development/tools/gb-studio/default.nix

```
@@ -0,0 +1,48 @@
+{ lib, appimageTools, fetchurl, makeDesktopItem }:
+
+appimageTools.wrapType2 rec {
+  pname = "gb-studio";
+  version = "3.1.0";
+
+  src = fetchurl {
+    url =
+      "https://github.com/chrismaltby/${pname}/releases/download/v${version}/${pname}-linux.AppImage";
+    hash = "sha256-vHk7rDbQsU+qPSdhH/34OTvS4jG5ybyPEr9ZTl4ZUxs=";
+  };
+
+  runScript = "appimage-exec.sh -w ${
+      appimageTools.extractType2 { inherit pname version src; }
+    } -- --disable-seccomp-filter-sandbox";
+
+  extraInstallCommands = let
+    desktopItem = makeDesktopItem {
+      name = pname;
+      desktopName = "GB Studio";
+      comment = "Visual retro game maker";
+      genericName = "GB Studio";
+      exec = "${meta.mainProgram} %U";
+      icon = pname;
+      type = "Application";
+      startupNotify = true;
+      categories = [ "GNOME" "GTK" "Utility" ];
+    };
+    # bash
+  in ''
+    mv "$out/bin/${pname}-${version}" "$out/bin/${pname}"
+    mkdir -p $out/share/applications
+    cp ${desktopItem}/share/applications/*.desktop $out/share/applications
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```

tracking issue: #292468